driver-core/src/main/com/mongodb/internal/session/BaseClientSessionImpl.java

```diff
@@ -24,7 +25,7 @@
 
 import static com.mongodb.assertions.Assertions.isTrue;
 
-public class ClientSessionImpl implements ClientSession {
+public class BaseClientSessionImpl implements ClientSession {
```

Comment:
Since it's internal, renamed to avoid having a class in high with the same name extending this one.